addons/mass_mailing/static/src/builder/options/snippet_visibility_option.inside.scss

```diff
@@ -0,0 +1,12 @@
+section[data-filter-domain] {
```

Comment:
idea out of nowhere: we should maybe hide the watermark on hover, or when the element is selected (with a transition?), in case it is in the way of content that the user wants to see?